source/includes/language-compatibility-table-ruby.rst

```diff
@@ -0,0 +1,222 @@
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility-large no-padding
+
+   * - Ruby Driver
+     - Ruby 3.2
+     - Ruby 3.1
+     - Ruby 3.0
+     - Ruby 2.7
+     - Ruby 2.6
+     - Ruby 2.5
+     - Ruby 2.4
+     - Ruby 2.3
+     - Ruby 2.2
+     - Ruby 2.1
+     - Ruby 2.0
+     - Ruby 1.9
```

Comment:
I question the utility of including these ancient Ruby versions in the compatibility chart. To me, it gives the impression that we endorse the use of old Rubies and old driver versions, which I don't think we really do. We don't really offer any support for folks who run into issues with older versions of our software.

/cc @alexbevi -- do you have any thoughts on this?

Reply:
In other standardization updates like this, I think we opted to remove old version from the table so I'm happy to do so here. Will see how @alexbevi weighs in.

Reply:
Let's remove everything older than MongoDB 4.0 from that matrix. Good call out Jamis!